aws/src/main/java/org/apache/iceberg/aws/s3/VendedCredentialsProvider.java

```diff
@@ -74,7 +75,11 @@ private RESTClient httpClient() {
     if (null == client) {
       synchronized (this) {
         if (null == client) {
-          client = HTTPClient.builder(properties).uri(properties.get(URI)).build();
+          client =
+              HTTPClient.builder(properties)
+                  .uri(properties.get(URI))
+                  .withAuthSession(AuthSession.EMPTY)
```

Comment:
Same here?  The default should not be EMPTY, that just cases it to fall through.